src/main/kotlin/AssertkMigratorMain.kt

```diff
@@ -37,11 +43,11 @@ private class AssertkMigratorCommand : CliktCommand(name = "assertk-migrator") {
 	override fun run() {
 		for (projectDir in projectDirs) {
 			projectDir.walk()
-				.filter { ".git/" !in it.path }
-				.filter { "/build/" !in it.path }
-				.filter(File::isFile)
+				.filter { ".git/" !in it.toString() }
+				.filter { "/build/" !in it.toString() }
+				.filter(Path::isRegularFile)
 				.forEach { file ->
-					if (file.name.endsWith(".kt")) {
+					if (file.nameWithoutExtension == "kt") {
```

Comment:
This doesn't seem right. We're specifically looking at the extension not the name.

Reply:
Ooops, a typo here. I wanted to type `extension`.